### ISSUE TITLE
Prision Stations

### ISSUE THREAD

**Issue by sqrw** [sqrw/sqrw.github.io#36]
This is a dublicate of Jail & prisons

**Comment by StarCitizenTracker** [comment]
A dedicated station that functions as a prison would require unique architecture one would think.

**Comment by sqrw** [comment]
Good point while related it's probably worth tracking seperately, while a prison station would imply prisons, prisons don't imply a prison station.
